bindings/pyroot_experimental/PyROOT/ROOT/setup.py.in

```diff
@@ -0,0 +1,9 @@
+from distutils.core import setup
+
+setup(name = 'ROOT',
+      version = '${ROOT_VERSION}',
+      packages=['ROOT', 'ROOT.pythonization'],
+      package_dir={'' : '${CMAKE_CURRENT_SOURCE_DIR}/python'},
+      url = 'https://root.cern.ch/pyroot',
```

Comment:
You can probably avoid having this as a .in configure file by using a relative path for `package_dir`. Also, these days we prefer `https://root.cern`.